### PR TITLE
DB privs to run scripts in "Identifying and Removing Corrupt Data" article.

### PR DESCRIPTION
**Pull request by allipatev**

### CODE REVIEW

File: Database-Features/removing-corrupt-data.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ set verbose on;
 @/tmp/check_all_columns.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. 
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. Database user requires SELECT ANY DICTIONARY system privilege for executing this script, see [Details on Rights Management](https://docs.exasol.com/db/latest/database_concepts/privileges/details_rights_management.htm).
```
Comment (nconforti93) [inline Database-Features/removing-corrupt-data.md:48]: @allipatev the user also needs the permission to SELECT every table in the database. I would rather state that the user must be a DBA for simplification.

File: Database-Features/removing-corrupt-data.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ set verbose on;
 @/tmp/enforce_indexes.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. Database user requires privileges to create and drop indices to run this script. DBA role would be sufficient, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles). You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
```
Comment (nconforti93) [inline Database-Features/removing-corrupt-data.md:80]: For simplification, I would also put here that it requres DBA privileges, no need to mention something specific about creating and dropping indices

**Review by nconforti93 — CHANGES_REQUESTED** [review]

File: Database-Features/removing-corrupt-data.md (hunk 1 above)
Comment (allipatev) [reply Database-Features/removing-corrupt-data.md]: Agree, it seemed obvious to me yesterday ...

**Comment by allipatev** [comment]
Thanks @nconforti93 !
I made changes and requested your review again.

File: Database-Features/removing-corrupt-data.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ set verbose on;
 @/tmp/check_all_columns.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. 
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. DBA role would be sufficient for DB user to execute this script, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles).
```
Comment (nconforti93) [inline Database-Features/removing-corrupt-data.md:48]:
```suggestion
To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. The user running this script requires the DBA role, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles) for more information.
```

File: Database-Features/removing-corrupt-data.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ set verbose on;
 @/tmp/enforce_indexes.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. DBA role would be sufficient for DB user to execute this script, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles). You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
```
Comment (nconforti93) [inline Database-Features/removing-corrupt-data.md:80]:
```suggestion
To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. The user running this script requires the DBA role, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles) for more information. You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
```

**Review by nconforti93 — CHANGES_REQUESTED** [review]

**Comment by allipatev** [comment]
Thanks @nconforti93 !
I made changes and requested your review again.

**Review by nconforti93 — APPROVED** [review]

**Pull request merged**